generate_client.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+### Run like so:
+# FILE=openapi.json ./generate_client.sh
+# the openapi schema needs to be in this directory for the container to see it.
+
+docker run -it --rm \
+    -v /tmp:/tmp \
+    -v $PWD:/workdir \
+    openapitools/openapi-generator-cli \
+    generate -i /workdir/$FILE -g ruby -o /tmp/test -c /workdir/config.json
+
+echo -e "=======
+Client generated from $FILE in: /tmp/test
```

Comment:
Could we set the `-o /tmp/test` to be a variable in the script? I know I like to output it in my current local cloned location and then push it up so my local repo always has a fresh client.

Reply:
Done!